modules/andes-core/common/src/main/java/org/wso2/andes/util/Strings.java

```diff
@@ -40,13 +41,10 @@
 
     private static final byte[] EMPTY = new byte[0];
 
-    private static final ThreadLocal<char[]> charbuf = new ThreadLocal<char[]>()
-    {
-        public char[] initialValue()
-        {
-            return new char[4096];
-        }
-    };
+    /**
+     * Resolving the UTF_8 charset
+     */
+    private static final Charset UTF_8_CHARSET = Charset.forName("UTF-8");
```

Comment:
Yes I saw this. I had in mind that we need to support JDK6. Can use this for the public branch since theres no need to support JDK 6. Thanks for pointing this out.